mne/viz/backends/_pyvista.py

```diff
@@ -825,9 +831,14 @@ def _take_3d_screenshot(figure, mode='rgb', filename=None):
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", category=FutureWarning)
         _process_events(figure.plotter)
-        return figure.plotter.screenshot(
+        # FIX: https://github.com/pyvista/pyvista/pull/995
+        old_window_size = figure.plotter.window_size
+        figure.plotter.window_size = figure.store["window_size"]
+        img = figure.plotter.screenshot(
             transparent_background=(mode == 'rgba'),
             filename=filename)
+        figure.plotter.window_size = old_window_size
```

Comment:
try/except, or not necessary here?

Reply:
For `window_size`? It exists now for a while. I don't think it's necessary.

Reply:
I mean if `screenshot` throws an error then` plotter.window_size` is never set back to its own value. So:
```
figure.plotter.window_size = ...
try:
    img = ...
finally:
    figure.plotter.window_size = old_window_size
```